R/get_nested_model_coefficients.R

```diff
@@ -40,10 +40,7 @@ get_nested_model_coefficients <- function(model,
       # If parameters::model_parameters wasn't implemented for the model type
       # let's grab the coefficients manually if possible
 
-      if (grepl("Error: No tidy method for objects of class",
-        as.character(e),
-        ignore.case = TRUE
-      )) {
+      if (grepl("error", as.character(e), ignore.case = TRUE)) {
```

Comment:
Do you have a more specific error message? `error` is very generic, and if it's a different error than "model not supported", we might need to throw a warning for it, so it can be fixed.

Reply:
It seems that `parameters` doesn't check whether a model is supported but just throws a random error? Is there any way, you could have more meaningful error messages? Here's the svm model from previously:

```r
> parameters::model_parameters(model = m)
Error in if (column_index > ncol(cs)) { : argument is of length zero
```
It would be better to get `the model is not of a supported type` or something like that?

Reply:
Hmm, well, the problem is that if `parameters` doesn't find an associated method, it uses `default` method. And, so, the error is going to relate to the exact step in the default method where it will falter. 

In other words, the exact error description will change on the unsupported object-basis.

Reply:
Perhaps wrap the call, catch the error and then make it more informing? I see that it can be difficult to take all model types into consideration though!

Reply:
Although it would only be in the default/fallback case, it would be needed right? And there you would know, that it could be because the model object is not supported, so a somewhat (more) meaningful error message would be possible to construct :)

Reply:
Hey, haven't forgotten about this. Just discussing this with my `parameters` team. Will get back to you after that.

That said, this PR will have to wait to be merged after the next release of `parameters`, which will be in a couple of weeks or so.

Reply:
I will go ahead and release my current changes then. Then this can get merged in a few weeks, when it's ready :-)